usage-examples/go/atlas-sdk-go/README.md

```diff
@@ -0,0 +1,90 @@
+# Atlas SDK for Go
+
+This project demonstrates how to script specific functionality using the Atlas
+SDK for Go. Code examples are used in the Atlas Architecture Center docs, and
+the project is made available in a user-facing repo.
+
+## Project Structure
+```text
+atlas-sdk-go/
+│── cmd/                  # Self-contained, runnable scripts
+│   ├── get_logs/  
+│       ├── main.go             
+│   ├── get_metrics_disk/
+│       ├── main.go
+│   ├── get_metrics_process/
+│       ├── main.go
+│── config/                # Atlas configuration settings
+│   ├── config.json             
+│── internal/              # Shared internal logic
+│   ├── auth/
+|       ├── client.go
+│   ├── config/
+|       ├── json.go
+|       ├── secrets.go
+|       ├── loader.go
+│── .env                   # Secrets file (excluded from Git)
+│── go.mod                     
+│── go.sum                           
+│── README.md              # Internal-only README (do not copy with Copier Tool)
+│── scripts/               # Internal-only Bluehawk scripts to snip and copy code examples
+│   ├── bluehawk.sh
+```
+
+## Runnable Scripts
+You can run individual scripts from the terminal. For example, to run `get_logs/main.go`:
+```shell
+go run cmd/get_logs/main.go
+```
+
+## Set up 
```

Comment:
Just a Q - should this `Set up` section also be in the public README? Seems it would be useful to give folks info about the service account prereq, and info about where/how to set env variables and config file information.